ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/FieldTemplate.js

```diff
@@ -1,12 +1,11 @@
 import React from "react";
 
 export const CustomFieldTemplate=(props) => {
-  const {  classNames, help,  description,  children } = props;
```

Comment:
the reason to remove help and description,because in the UI we are handling it differently.

Reply:
So we donot need the description provided by rjsf.

Reply:
Makes sense.